python/ray/tests/test_metrics_agent.py

```diff
@@ -75,6 +74,9 @@
     "ray_gcs_actors_count",
 ]
 
+if not ray._raylet.Config.use_ray_syncer():
```

Comment:
Why not just supporting this from the syncer?

Reply:
I'd probably prefer another name instead of `ray_outbound_heartbeat_size_kb_sum`